resources/icons/16px/app-header.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>bp3-icon-app-header</title>
+    <defs>
+        <path d="M15,0 C15.5522847,0 16,0.44771525 16,1 L16,15 C16,15.5522847 15.5522847,16 15,16 L1,16 C0.44771525,16 0,15.5522847 0,15 L0,1 C0,0.44771525 0.44771525,0 1,0 L15,0 Z M6,4 C6,3.44771525 5.55228475,3 5,3 C4.44771525,3 4,3.44771525 4,4 L4,4 L4,12 C4,12.5522847 4.44771525,13 5,13 C5.55228475,13 6,12.5522847 6,12 L6,12 L6,9 L10,9 L10,12 C10,12.5522847 10.4477153,13 11,13 C11.5522847,13 12,12.5522847 12,12 L12,12 L12,4 C12,3.44771525 11.5522847,3 11,3 C10.4477153,3 10,3.44771525 10,4 L10,4 L10,7 L6,7 Z" id="path-1"></path>
+    </defs>
+    <g id="bp3-icon-drawer-app-header" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <mask id="mask-2" fill="white">
```

Comment:
these icons are broken because of the way they were exported from Sketch. please try to export the simplest possible shape so that this SVG looks closer to the other SVGs in this folder. you are having an issue similar to https://github.com/palantir/blueprint/issues/4102, which was fixed by #4104